base/system/diskpart/lang/tr-TR.rc

```diff
@@ -114,12 +122,12 @@ END
 /* SELECT command strings */
 STRINGTABLE
 BEGIN
-    IDS_SELECT_NO_DISK "\nŞimdilik bir disk seçilmemiş.\nLütfen bir disk seçiniz ve yeniden deneyiniz.\n\n"
+    IDS_SELECT_NO_DISK "\nBir disk seçilmemiş.\nLütfen bir disk seçiniz ve yeniden deneyiniz.\n\n"
```

Comment:
May be you can say “Henüz bir disk seçilmemiş” for it.

Reply:
I do prefer the shorter version actually that was chosen here by @vsaucing . Especially since it has not been "backed up" yet in any older branch, for the history of mankind.

Reply:
> May be you can say “Henüz bir disk seçilmemiş” for it.

nah this is better